android/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
   }
 
   dependencies {
-    classpath "com.android.tools.build:gradle:7.2.1"
+    classpath "com.android.tools.build:gradle:7.3.1"
```

Comment:
Normally we don't bump any dependencies and compile versions etc inside the `android` directory of library since it is never used. You never run your library directly, only as a dependency. e.g. [here](https://github.com/software-mansion/react-native-screens/blob/4d2d4172626727c6eb01fc21d7f679041cda8365/android/build.gradle#L6) we got some old options. You can keep those changes though, just saying.